plugins/modules/device.py

```diff
@@ -290,6 +296,7 @@ def main():
             vc_priority=dict(required=False, type="int"),
             comments=dict(required=False, type="str"),
             local_config_context_data=dict(required=False, type="dict"),
+            controller_managed_device_group=dict(required=False, type="raw"),
```

Comment:
Can you please add this to one of the existing [integration tests](https://github.com/nautobot/nautobot-ansible/blob/develop/tests/integration/targets/latest/tasks/device.yml) for device? You can follow this pattern to omit the field for the unsupported versions:
https://github.com/nautobot/nautobot-ansible/blob/fe3f4f7ef6ee48bb55da657599b4e7f15bf2182c/tests/integration/targets/latest/tasks/location_type.yml#L59